### PR TITLE
Add gleam programming language

### PR DESCRIPTION
**Pull request by zunda-arrow**
Checklist:

-   [x] The package builds locally with `./piston build-pkg [package] [version]`
-   [x] The package installs with `./piston ppman install [package]=[version]`
-   [x] The package runs the test code with `./piston run [package] -l [version] packages/[package]/[version]/test.*`
-   [x] Package files are placed in the correct directory
-   [x] No old package versions are removed
-   [x] All source files are deleted in the `build.sh` script
-   [x] `metadata.json`'s `language` and `version` fields match the directory path
-   [x] Any extensions the language may use are set as aliases
-   [x] Any alternative names the language is referred to are set as aliases.

Im working on adding gleam. For some reason it won't allow me to install the package after running `build-pkg` so I can't install. I need help because I'm not sure  why it isn't working.

### CODE REVIEW

File: packages/gleam/0.26.2/build.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Install Erlang
+apt-get update
+apt-get install wget -y
+wget https://packages.erlang-solutions.com/erlang-solutions_2.0_all.deb
+dpkg -i erlang-solutions_2.0_all.deb
+apt-get install esl-erlang
```
Comment (Enderchief) [inline packages/gleam/0.26.2/build.sh]:
```suggestion

# Install Erlang
source ../../erlang/23.0.0/build.sh
```

guaranteed to build erlang

File: packages/gleam/0.27.0/run [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+shift
+cd project
+gleam run $(cat ../compile_args 2> /dev/null) -- "$@"
```
Comment (zunda-arrow) [inline packages/gleam/0.27.0/run:5]: Wondering if it makes sense to discard the three first lines of stdout since they don't matter when using piston

Comment (Enderchief) [reply packages/gleam/0.27.0/run:5]: Honestly I have not idea

**Comment by Enderchief** [comment]
LGTM :+1:

File: packages/gleam/0.27.0/run (hunk 2 above)
Comment (zunda-arrow) [reply packages/gleam/0.27.0/run:5]: Im making a PR for it that can be merged if the gleam output seems weird

**Pull request merged**